packages/service/src/ocr/utils/ocrGetTextPositions.ts

```diff
@@ -0,0 +1,18 @@
+import ocrGetData from './ocrGetData.js'
```

Comment:
I think we can remove the `ocr` in the file name since all these files are in an `ocr` directory.

Reply:
Thanks, good suggestion